app/steps/ui/payment/breakdown/index.js

```diff
@@ -50,6 +50,14 @@ class PaymentBreakdown extends Step {
 
     * handlePost(ctx, errors, formdata, session, hostname) {
         if (formdata.paymentPending !== 'unknown') {
+            if (!formdata.applicantEmail) {
+                logger.warn('Unable to find applicantEmail, using session.regId instead.');
+                if (session.regId) {
+                    formdata.applicantEmail = session.regId;
+                } else {
+                    logger.error('Unable to find applicantEmail or session.regId.');
+                }
+            }
```

Comment:
Can we have a unit test to specifically test this new logic, so we can see that if the `formdata.applicantEmail` doesn't exist then it's retrieved from `session.regId`